test/tests/functional/pbs_qmgr.py

```diff
@@ -0,0 +1,128 @@
+        :type line: String
+        """
+        if line is None:
+            return
+
+        if line[0] == " ":
+            # Spaces are prefix for new attribute lines
+            # Make sure that this is a new attribute line
+            self.assertTrue("=" in line)
+        elif line[0] == "\t":
+            # Tabs are prefix for line extensions
+            # Make sure that this is a line extension
+            self.assertTrue("=" not in line)
+
+    def test_listcmd_whitespaces(self):
+        """
+        Check that the prefix for new attributes listed out by qmgr list
+        are spaces and that for line extensions is a tab
+        """
+        pass
+        _, fn = self.du.mkstemp()
+
+        # Check 1: New attributes are prefixed with spaces and not tabs
+        # Execute qmgr -c 'list sched' and store output in a temp file
+        if self.du.is_localhost(self.server.hostname) is True:
+            qmgr_cmd = ["qmgr", "-c", "list sched"]
+        else:
+            qmgr_cmd = ["qmgr", "-c", "\'list sched\'"]
+        with open(fn, "w+") as tempfd:
+            ret = self.du.run_cmd(self.server.hostname, qmgr_cmd,
+                                  stdout=tempfd)
+            self.assertTrue(ret['rc'] == 0)
+            for line in tempfd:
+                self.check_whitespace_prefix(line)
+
+        # Check 2: line extensions are prefixed with tabs and not spaces
+        # Create a random long, comma separated string
+        blah = "blah"
+        long_string = ""
+        for i in range(49):
+            long_string += blah + ","
+        long_string += blah
+        # Create a new vnode
+        attrs = {"resources_available.ncpus": 2}
+        vndef = self.mom.create_vnode_def("vn", attrs)
+        vndef_file = "tempdef"
+        self.mom.insert_vnode_def(vndef, fname=vndef_file)
+        # create_vnode_def added "[0]" to the vnode name passed to it
+        nodename = "vn[0]"
+        cmd = ["pbsnodes", "-av"]
+        self.du.run_cmd(cmd=cmd)
+        # Set 'comment' attribute to the long string we created above
+        attrs = {"comment": long_string}
+        self.server.manager(MGR_CMD_SET, VNODE, attrs, nodename)
+        # Execute "qmgr 'list node vn[0]'"
+        # The comment attribute should generate a line extension
+        if self.du.is_localhost(self.server.hostname) is True:
+            qmgr_cmd = ["qmgr", "-c", "list node " + nodename]
+        else:
+            qmgr_cmd = ["qmgr", "-c", "\'list node " + nodename + "\'"]
+        with open(fn, "w+") as tempfd:
+            ret = self.du.run_cmd(self.server.hostname, qmgr_cmd,
+                                  stdout=tempfd)
+            self.assertTrue(ret['rc'] == 0)
+            for line in tempfd:
+                self.__check_whitespace_prefix(line)
+
+        # Cleanup
+        # Remove the temporary file
+        os.remove(fn)
+        # Delete the vnode created
+        self.mom.delete_vnode_defs(vndef_file)
```

Comment:
Actually this would leave the vnode as it as when I'd run this test. Maybe it's because this suite has only one test case so setup() doesn't get called again to clean stuff up.